versions.yaml

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: |
+        This file contains version details that are used by various
+        repositories for setting up the correct environment to run
+        tests and package components.
+
+        Notes:
+
+        - All sections *MUST* include a URL where applicable.
+
+        - All sections *SHOULD* include a description where intent is
+          not clear.
+
+        - WARNING: Gotcha alert! Remember to double-quote all versions
```

Comment:
Maybe you should double quote every version in this file ? No confusion this way

Reply:
Good call. It might be overkill, but I've gone for the nuclear option - all strings are quoted for maximum safety.